source/documentation/virtual_machine_management_guide/topics/Virtual_Machine_Resource_Allocation_settings_explained.adoc

```diff
@@ -32,9 +32,14 @@ a|Allows users to set the level of CPU resources a virtual machine can demand re
 
 * *Custom* - A custom level of CPU shares defined by the user.
 | No.
+| |*CPU Pinning Policy*
+a|
 
+* `None` - runs without any CPU pinning.
 
-|
+* `Resize and Pin NUMA` - resizes the virtual CPU and NUMA topology of the virtual machine according to the Host, and pins them to the Host resources.
```

Comment:
there are more policies here, dedicated, isolate threads and manual (to use the manual pinning described below).

Reply:
I see those additional options, but they look like they are disabled - greyed out. 
Can I list them all as they they show up in the drop-down list?